api/src/controllers/user.js

```diff
@@ -59,23 +49,15 @@ router.get(
     });
     if (!user) return res.status(404).json({ ok: false, error: "user not found" });
 
-    const xforwarded = await getLocationFromIp(req.headers["x-forwarded-for"]);
-    const remote = await getLocationFromIp(req.connection?.remoteAddress);
-    const socket = await getLocationFromIp(req.socket?.remoteAddress);
-    const connectionSocket = await getLocationFromIp(req.connection.socket?.remoteAddress);
-    const ip = await getLocationFromIp(req.ip);
-
-    const ips = {
-      xforwarded,
-      remote,
-      socket,
-      connectionSocket,
-      ip,
-    };
+    const xforwarded = geoip.lookup(req.headers["x-forwarded-for"]);
+    if (xforwarded) {
+      const { region } = xforwarded;
+      isWellLocated = region === "IDF";
+    }
```

Comment:
fais attention aux noms de variable


```suggestion
    const xforwarded = req.headers["x-forwarded-for"];
    const location = geoip.lookup(xforwarded);
    if (location) {
      const { region } = location;
      isWellLocated = region === "IDF";
    }
```